rust/opendp-ffi/src/trans/bootstrap.json

```diff
@@ -427,6 +427,75 @@
         ],
         "ret": {"c_type": "FfiResult<AnyTransformation *>"}
     },
+    "make_find": {
+        "description": "Find the index of a data value in a set of categories.",
+        "features": ["contrib"],
+        "args": [
+            {
+                "name": "categories",
+                "c_type": "AnyObject *",
+                "rust_type": {
+                    "origin": "Vec",
+                    "args": ["TIA"]
+                },
+                "description": "The set of categories to compute counts for."
+            },
+            {
+                "name": "TIA",
+                "is_type": true,
+                "description": "categorical/hashable input type. Input data must be Vec<TIA>."
+            }
+        ],
+        "ret": {"c_type": "FfiResult<AnyTransformation *>"}
+    },
+    "make_find_bin": {
+        "description": "Find the bin index from a monotonically increasing vector of edges.",
+        "features": ["contrib"],
+        "args": [
+            {
+                "name": "edges",
+                "c_type": "AnyObject *",
+                "rust_type": {
+                    "origin": "Vec",
+                    "args": ["TIA"]
+                },
+                "description": "The set of edges to split bins by."
+            },
+            {
+                "name": "TIA",
+                "is_type": true,
+                "description": "categorical/hashable input type. Input data must be Vec<TIA>."
```

Comment:
It's so easy to forgot the documentation! Will be more careful about that in the future. I also loosened the categories argument to make_index, to be any primitive type, instead of any hashable type.